src/Microsoft.Azure.SignalR.Management/ServiceManagerBuilder.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.Azure.SignalR.Management
     /// <summary>
     /// A builder for configuring <see cref="IServiceManager"/> instances.
     /// </summary>
+    [Obsolete("Use ServiceHubContextBuilder to build ServiceHubContext directly.")]
```

Comment:
add a sample showing how to convert to use `ServiceHubContextBuilder` here?

Reply:
Maybe we also need a complete document showing how to migrate to the new pattern